mne/viz/topomap.py

```diff
@@ -100,9 +142,6 @@ def _prepare_topomap_plot(inst, ch_type, layout=None, sphere=None):
         elif ch_type == 'csd':
             picks = pick_types(info, meg=False, csd=True, ref_meg=False,
                                exclude='bads')
-        elif ch_type in ['hbo', 'hbr', 'fnirs_raw', 'fnirs_od']:
```

Comment:
I would make a `_fnirs_types = ('hbo', 'hbr', 'fnirs_raw', 'fnirs_od')` somewhere above outside this function and reuse it here, above, below, and in `plot_evoked_topomap`.